src/recipes/lib/authz.js

```diff
@@ -0,0 +1,105 @@
+import * as request from './request';
+import { webtaskRegion } from './webtask';
+
+const REQUIRED_ACCESS_TOKEN_SCOPES = [
+  'read:groups', 'create:groups', 'update:groups',
+  'read:roles', 'create:roles', 'update:roles',
+  'read:permissions', 'create:permissions'
+];
+
+export function apiBaseUrl () {
+  return `https://${process.env.AUTH0_TENANT}.${webtaskRegion()}.webtask.io/adf6e2f2b84784b57522e3b19dfc9201/api`;
```

Comment:
@saltukalakus: Good point. While that ID never changes, it's still good to put it somewhere more configurable. We'll need to add it as a value in the `.auth0-reset-tenant-env` file described in the README [here](https://github.com/twistedstream/auth0-reset-tenant/blob/authz-api-support/README.md#global-setup)